src/cmd/cli/command/commands.go

```diff
@@ -1127,29 +1132,29 @@ func getProvider(ctx context.Context, loader cliClient.Loader) (cliClient.Provid
 		return nil, err
 	}
 
-	projName, err := cliClient.LoadProjectNameWithFallback(ctx, loader, provider)
+	return provider, nil
+}
+
+func canIUseProvider(ctx context.Context, provider cliClient.Provider, projectName string) (cliClient.Provider, error) {
+	cdImage, err := allowToUseProvider(ctx, providerID, projectName)
```

Comment:
Is there still a point to have separate `canIUseProvider` and `allowToUseProvider`? I can't tell from the name what one does vs the other.